### ISSUE TITLE
Download lower-res version without EXIF data, i.e., thumbnail

### ISSUE THREAD

**Issue by reox** [pulsejet/memories#1359]
**Is your feature request related to a problem? Please describe.**
I often need to upload photos somewhere or share them via mail. I do not need and want to share the full-res version including the EXIF data. Thus, a button to download a lower-res version (i.e., 1500px × 1500px) without EXIF data (especially location information) would be nice. 

**Describe the solution you'd like**
I guess these images are already available as the thumbnails (I checked, they do not contain any EXIF data): Thus, memories could just add a button to download a specific thumbnail. 

**Describe alternatives you've considered**
Memories could also create a specific sized image on demand.

**Comment by pulsejet** [comment]
You can "share" the reduced size thumbnail, this has no exif data.

![Image](https://github.com/user-attachments/assets/3c36c61c-6932-4219-bfe7-e83a7a226114)

**Comment by reox** [comment]
But this is only available in the App - is that correct? In the browser I only see the "Public Link" Button when clicking on Share.
But that would be exactly what I need!

**Comment by pulsejet** [comment]
Maybe your share api is disabled, are you on HTTPS?

**Comment by reox** [comment]
yes - however the nextcloud instance is behind a reverse proxy and between them there is only unencrypted traffic

**Comment by reox** [comment]
interesting. I'm not in icognito but perhaps I set some setting somewhere in Firefox to prevent that. Indeed, it works in Chrome.
So I'll close that because apparently it is already implemented and just my browser is not showing it!
